packages/e2e/test/k6/scenarios/README.md

```diff
@@ -0,0 +1,21 @@
+# Running tests locally
```

Comment:
better to move it one level higher to `packages/e2e/test/k6`

Reply:
done here: [6aa3d32](https://github.com/input-output-hk/cardano-js-sdk/pull/808/commits/6aa3d326fccf6275d16b75da47b0d0229a8f0916)